### PR TITLE
Import Rust packages

### LINKED ISSUE #551: Import Rust packages
Currently, Rust (crates.io) packages are only installed, but not imported. We should import them as well.

Carried over from #22

### PR DESCRIPTION
**Pull request by dukecat0**
Closes #551

### CODE REVIEW

File: sandboxes/crates.io/analyze.py [hunk 1]
```diff
@@ -38,10 +40,25 @@ def install(package: Package):
...
 
+def importPkg(package: Package):
+    path_to_rs = os.path.join(os.getcwd(), 'src', 'main.rs')
+    try:
+      with open(path_to_rs, 'r+') as handle:
+        content = handle.read()
+        handle.seek(0, 0)
+        handle.write('#[allow(unused_imports)]\n')
+        handle.write(f'use {package.name.strip()}::*;' + '\n' + content)
+        handle.flush()
```
Comment (calebbrown) [inline sandboxes/crates.io/analyze.py:51]: Is `./src/main.rs` likely to exist that we need to prepend the file rather than just replacing it?

**Review by calebbrown — CHANGES_REQUESTED** [review]
Thanks for adding this! Very much appreciated.

I have added one comment that applies to both files.

One other thing that needs to be done is updating the configuration in `internal/pkgmanager/crates.io.go` to set `dynamicPhases: analysisrun.DefaultDynamicPhases()` in the structs at the bottom.

File: sandboxes/crates.io/analyze.py (hunk 1 above)
Comment (dukecat0) [reply sandboxes/crates.io/analyze.py:51]: It's created when `cargo init` was run here: https://github.com/ossf/package-analysis/blob/07a32fcad24c01693ebb779cb571480b6ac8cade/sandboxes/dynamicanalysis/Dockerfile#L127

Comment (calebbrown) [reply sandboxes/crates.io/analyze.py:51]: Thanks!

**Review by calebbrown — APPROVED** [review]

**Pull request merged**